libs/sarif/src/schema/log.rs

```diff
@@ -0,0 +1,66 @@
+use alloc::{borrow::Cow, vec::Vec};
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+use crate::schema::{Run, SchemaVersion, SCHEMA_ID};
+
+/// Static Analysis Results Format (SARIF) Version 2.1.0 JSON Schema: a standard format for the
+/// output of static analysis tools.
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(
+    feature = "serde",
+    derive(Serialize, Deserialize),
+    serde(rename_all = "camelCase")
+)]
+pub struct Log {
```

Comment:
Spec names this `SarifLog`, I am fine with keeping this `Log` tho

Reply:
I'd prefer `SarifLog`, I'm going to change it